tests/golden-layout.spec.ts

```diff
@@ -26,9 +26,37 @@ test.describe('tests golden-layout operations', () => {
   });
 
   test.afterAll(async () => {
-    //  reset layout
-    await page.getByTestId('app-main-panels-button').click();
-    await page.getByLabel('Reset Layout').click();
+    /**
+     * Open panels menu, reset layout, confirm or cancel "Reset Layout" prompt
+     */
+    async function resetLayout(confirm: boolean) {
+      await page.getByTestId('app-main-panels-button').click();
+      await page.getByLabel('Reset Layout').click();
+
+      if (confirm) {
+        await page
+          .locator('.modal .btn-danger')
+          .filter({ hasText: 'Reset' })
```

Comment:
```suggestion
          .getByRole('button', { name: 'Reset' })
```

Reply:
This doesn't seem to work. These buttons don't have "role" set.

Reply:
Weird. Or just a missing attribute on our part

Reply:
Yeah, probably could add role attributes, but not really wanting to rock the boat for this PR in case it impacts snapshots in other tests or something